data/json/items/chemicals_and_resources.json

```diff
@@ -890,5 +890,24 @@
     "weight": 212,
     "volume": 1,
     "to_hit": -2
-    }
+    },{
+    "type" : "COMESTIBLE",
+    "id" : "sewage_water",
```

Comment:
Yes, I noticed that also but it appears to be the exception and in general hierarchical names are best with the most specific part first